scripts/verify_tdr_sources.py

```diff
@@ -93,9 +93,9 @@ def verify_source(self,
                       ) -> None:
         source = self.tdr.lookup_source(source_spec)
         log.info('TDR client is authorized for API access to %s.', source_spec)
-        require(source.project == source_spec.project,
+        require(source_spec.project in (source.project, config.google_project()),
                 'Actual Google project of TDR source differs from configured one',
-                source.project, source_spec.project)
+                source.project, source_spec.project, config.google_project())
```

Comment:
```suggestion
                source_spec.project, source.project, config.google_project())
```

Reply:
I decided on a different approach here to make the diff clearer and hopefully make the intent more apparent.